src/library/scala/collection/package.scala

```diff
@@ -106,13 +73,6 @@ package object collection extends LowPriority {
     }
   }
 
-  def optionToIterableOnce[A](maybeA: scala.Option[A]): IterableOnce[A] =
-     new Iterator[A] {
-       private var _hasNext = maybeA.nonEmpty
-       def next(): A = if (_hasNext) { _hasNext = false; maybeA.get } else Iterator.empty.next()
-       def hasNext: Boolean = _hasNext
-     }
```

Comment:
That one is useful to support `flatMap`ping to an `Option`

Reply:
There's a conversion to `Iterable` in the [`Option` companion](https://github.com/scala/scala/blob/v2.13.0-M4-pre-20d3c21/src/library/scala/Option.scala#L17). Perhaps that one should be replaced with the `IterableOnce` one?

Reply:
Hmm, it might be useful to keep both, at least for backward compatibility… The one that converts to `IterableOnce` only provides an `iterator()` operation, whereas the one that converts to `Iterable` provides all the operations of `Iterable`s.

Reply:
We considered the possibility to have `Option` extend `IterableOnce` directly. If we keep the existing implicit conversion I think it's fine to provide a full `Iterable`.